core-bundle/src/Resources/contao/library/Contao/InsertTags.php

```diff
@@ -170,7 +170,7 @@ protected function doReplace($strBuffer, $blnCache)
 
 				// Accessibility tags
 				case 'lang':
-					if (!$elements[1])
+					if (empty($elements[1]))
```

Comment:
IMHO this should be `$elements[1] ?? null` everywhere, so that the illegal array access is prevented but the logic stays the same. (And because using `empty` ist kind of an antipattern for nearly everything. :slightly_smiling_face:.)